lib/polymer/iron.dart

```diff
@@ -10,82 +10,26 @@ import 'dart:js';
 
 import 'base.dart';
 
-class CoreAnimatedPages extends CoreSelector {
-  CoreAnimatedPages() : super('core-animated-pages');
-  CoreAnimatedPages.from(HtmlElement element) : super.from(element);
+class IronPages extends IronSelectableBehavior {
+  IronPages() : super('iron-pages');
+  IronPages.from(HtmlElement element) : super.from(element);
 }
 
-// core-drawer-panel
-class CoreDrawerPanel extends CoreElement {
-  CoreDrawerPanel() : super('core-drawer-panel');
-  CoreDrawerPanel.from(HtmlElement element) : super.from(element);
-
-  void forceNarrow() => toggleAttribute('forceNarrow');
-
-  void makeDrawer(CoreElement element) =>
-      element.toggleAttribute('drawer', true);
-
-  void makeMain(CoreElement element) => element.toggleAttribute('main', true);
-
-  /// Toggles the panel open and closed.
-  void togglePanel() => call('togglePanel');
-
-  /// Opens the drawer.
-  void openDrawer() => call('openDrawer');
-
-  /// Closes the drawer.
-  void closeDrawer() => call('closeDrawer');
-}
-
-class CoreDropdown extends CoreOverlay {
-  CoreDropdown([String tag]) : super(tag == null ? 'core-dropdown' : tag);
-  CoreDropdown.from(HtmlElement element) : super.from(element);
-
-  String get halign => attribute('halign');
-  /// Accepted values: 'left', 'right'.
-  set halign(String value) => setAttribute('halign', value);
-
-  String get valign => attribute('valign');
-  /// Accepted values: 'top', 'bottom'.
-  set valign(String value) => setAttribute('valign', value);
-}
-
-class CoreHeaderPanel extends CoreElement {
-  CoreHeaderPanel() : super('core-header-panel');
-  CoreHeaderPanel.from(HtmlElement element) : super.from(element);
-}
-
-class CoreIcon extends CoreElement {
-  CoreIcon({String icon, String src}) : super('core-item') {
+class IronIcon extends PolymerElement {
+  IronIcon({String icon, String src}) : super('iron-icon') {
     if (icon != null) this.icon = icon;
     if (src != null) this.src = src;
   }
 
-  CoreIcon.from(HtmlElement element) : super.from(element);
+  IronIcon.from(HtmlElement element) : super.from(element);
 
   String get src => attribute('src');
   set src(String value) => setAttribute('src', value);
 }
 
-class CoreItem extends CoreElement {
-  CoreItem({String icon, String label}) : super('core-item') {
-    if (icon != null) this.icon = icon;
-    if (label != null) this.label = label;
-  }
-
-  CoreItem.from(HtmlElement element) : super.from(element);
-
-  void name(String value) => setAttribute('name', value);
-}
-
-class CoreMenu extends CoreSelector {
-  CoreMenu() : super('core-menu');
-  CoreMenu.from(HtmlElement element) : super.from(element);
-}
-
-class CoreOverlay extends CoreElement {
-  CoreOverlay([String tag]) : super(tag == null ? 'core-overlay' : tag);
-  CoreOverlay.from(HtmlElement element) : super.from(element);
+class IronOverlayBehavior extends PolymerElement {
+  IronOverlayBehavior([String tag]) : super(tag == null ? 'core-overlay' : tag);
```

Comment:
I wasn't completely sure what to do with this. Iron-overlay doesn't exist I think, you know have `behaviors` with a couple of methods,property attached to it, that you can implement, but not construct. You can't extend elements anymore in polymer 1.0. 

So there is now an iron-overlay tag, but you have `Polymer.IronOverlayBehavior`:

https://elements.polymer-project.org/elements/iron-overlay-behavior?active=Polymer.IronOverlayBehavior

I'm not really sure how to `translate` behaviors to dart.